docs/src/3_Automation/06_Bestiary.yml

```
@@ -1,171 +1,311 @@
-Name:                                 # must be unique
-  Type: Boss, Dealer, NPC, etc.       # Required field
-  HP: X                               # Required field
-  AR: X                               # Required field
-  PP: X                               # Required field
-  Skill Mods:                         # List only those that apply
-    AGL: ±X                           # + is implied if not explicit
-    CON: ±X
-    GUT: ±X
-    INT: ±X
-    STR: ±X
-    VIT: ±X
-  Threat Level: X                     # Consistent comparable rating fitting formula
-  Description: Descrip                # Full descriptive text
-
+Template:               # must be unique
+  Type: One of          # Dealer, Boss, NPC, Companion
+  Level: X              # Only for Dealers
+  Threat: X             # Measure of difficulty
+  HP: X                 # Required field
+  AP: X                 # 0 is default if blank
+  AR: X                 # Required field
+  PP: X                 # Required field
+  Attrib:               # List only Non-sero
+    AGL: ±X             # + is implied if not explicit
+    CON: ±X             #
+    GUT: ±X             #
+    INT: ±X             #
+    STR: ±X             #
+    VIT: ±X             #
+  Skills:               # List if different from corresponding Attrib
+    Finesse: ±X         #
+    Stealth: ±X         #
+    Bluffing: ±X        #
+    Performance: ±X     #
+    Knowledge: ±X       #
+    Investigation: ±X   #
+    Detection: ±X       #
+    Craft: ±X           #
+    Athletics: ±X       #
+    Brute: ±X           #
+  Powers:               # Include Power names as list
+  Phases:               # Only for bosses
+    One:                #
```

I think this came from my draft, but I'm thinking it should be an integer
```suggestion
    1:                  # Integer, phase number
```